src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudServiceActor.scala

```diff
@@ -76,11 +77,12 @@ class FireCloudServiceActor extends HttpServiceActor with FireCloudDirectives
   val trialProjectManager = system.actorOf(ProjectManager.props(app.rawlsDAO, app.trialDAO, app.googleServicesDAO), "trial-project-manager")
 
   if (logitMetricsEnabled) {
-    logger.info("Logit metrics are enabled.")
+    val freq = FireCloudConfig.Metrics.logitFrequencyMinutes
     val metricsActor = system.actorOf(MetricsActor.props(app), "metrics-actor")
     // use a randomized startup delay to avoid multiple instances of this app executing on the same cycle
     val initialDelay = 1 + scala.util.Random.nextInt(10)
-    system.scheduler.schedule(initialDelay.minutes, FireCloudConfig.Metrics.logitFrequencyMinutes.minutes, metricsActor, MetricsActor.RecordMetrics)
+    logger.info(s"Logit metrics are enabled: every $freq minutes, starting $initialDelay minutes from now.")
+    system.scheduler.schedule(initialDelay.minutes, freq.minutes, metricsActor, MetricsActor.RecordMetrics)
```

Comment:
I tweaked these log messages to make them consistent with the log messages below.